package.json

```diff
@@ -9,11 +9,11 @@
     "build": "run-s bootstrap build:**",
     "build:config": "node js/build-env-config.js > config/env.generated.json",
     "build:client": "run-s build-client:**",
-    "build-client:js": "webpack --config webpack.config.js --optimize-minimize --display-error-details --colors",
+    "build-client:js": "webpack --config webpack.config.js --display-error-details --colors",
     "build-client:scss": "node-sass scss/main.scss dist/css/main.css",
     "build-client:prefix:main": "postcss --use autoprefixer -o dist/css/main.css dist/css/main.css",
     "build-client:static": "run-s copy:**",
-    "build:server": "webpack --config webpack.server.config.js --optimize-minimize --display-error-details --colors",
```

Comment:
we only need to minimize things for for `production` and this is handled in the webpack config files.